### PR TITLE
chore: rename notice

### PR DESCRIPTION
**Pull request by Weakky**
- [x] update README
- [x] once 1.0 is cut for `nexus` then cut a `@nexus/schema@0.20` from this branch as well.
- [x] close, do not merge

image

### CODE REVIEW

File: scripts/rename-notice.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+process.stdout.write(`\
+${ansi.yellow(ansi.bold("WARNING: '@nexus/schema' will be renamed to 'nexus' on December 14th."))}
+${ansi.yellow(ansi.bold("WARNING: '@nexus/schema' will be renamed to 'nexus' on December 14th."))}
+${ansi.yellow(ansi.bold("WARNING: '@nexus/schema' will be renamed to 'nexus' on December 14th."))}
+
+0.20.x will be the last versions published on the '@nexus/schema' package.
+Future updates will be published to 'nexus@1.0.0'.
+`)
```
Comment (Weakky) [inline scripts/rename-notice.js:14]: Feel free to suggest a better wording

File: scripts/rename-notice.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  reset: '\u001b[0m',
+}
+
+process.stdout.write(`\
+${ansi.yellow(ansi.bold("WARNING: '@nexus/schema' will be renamed to 'nexus' on December 14th."))}
+${ansi.yellow(ansi.bold("WARNING: '@nexus/schema' will be renamed to 'nexus' on December 14th."))}
+${ansi.yellow(ansi.bold("WARNING: '@nexus/schema' will be renamed to 'nexus' on December 14th."))}
+
+0.20.x will be the last versions published on the '@nexus/schema' package.
+Starting December 14th, future updates will be published to 'nexus@^1.0.0'.
```
Comment (Weakky) [inline scripts/rename-notice.js:13]: Feel free to suggest a better wording

**Comment by jasonkuhrt** [comment]
whoops I clicked the wrong button, should not have merged trunk!

**Comment by Weakky** [comment]
>whoops I clicked the wrong button, should not have merged trunk! We'll need to revert that. Always forget how to revert merge commits...

Can't you just `git revert` or `git reset --hard <commit before merge> && git push --force` ?

**Comment by Weakky** [comment]
Reverted the merges

**Comment by jasonkuhrt** [comment]
https://www.npmjs.com/package/@nexus/schema/v/0.20.0
https://www.npmjs.com/package/@nexus/schema/v/0.20.1

image


image

**Pull request closed without merging**